powershell/plan.sh

```diff
@@ -0,0 +1,38 @@
+pkg_name="powershell"
+pkg_origin="core"
+pkg_version="6.1.0"
```

Comment:
We should use 6.2.1 to match the windows side.

Reply:
Thanks for catching this! Not sure where that version came from :smile: